runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,86 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (NULL != agent->dumpOptions)
+         {
+         char *indexOfDot = strchr(agent->dumpOptions, '.');
+         char *indexOfParen = strchr(agent->dumpOptions, '(');
+
+         if ((NULL != indexOfDot) && (NULL != indexOfParen))
+            {
+            TR::VMAccessCriticalSection findUserMethod(TR_J9VMBase::get(jitConfig, crashedThread));
+
+            char *className = agent->dumpOptions;
+            U_32 classNameLength = static_cast<U_32>(indexOfDot - agent->dumpOptions);
+            const char *methodName = indexOfDot + 1;
+            U_32 methodNameLength = static_cast<U_32>(indexOfParen - methodName);
+            const char *methodSig = indexOfParen;
+            U_32 methodSigLength = strlen(indexOfParen);
```

Comment:
suggest: `U_32 methodSigLength = strlen(methodSig);`

Reply:
Fixed in bfae119